CI failure TestUnwrapCastInComparison.testCastTimestampToTimestampWithTimeZone

This test is always failing now:
```
2020-11-29T04:47:26.6350565Z [ERROR] testCastTimestampToTimestampWithTimeZone(io.prestosql.sql.planner.TestUnwrapCastInComparison)  Time elapsed: 0.226 s  <<< FAILURE!
2020-11-29T04:47:26.6352965Z java.lang.IllegalStateException: missing expression for alias a
2020-11-29T04:47:26.6354928Z 	at io.prestosql.sql.planner.assertions.SymbolAliases.lambda$get$0(SymbolAliases.java:80)
2020-11-29T04:47:26.6356850Z 	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
2020-11-29T04:47:26.6358981Z 	at io.prestosql.sql.planner.assertions.SymbolAliases.get(SymbolAliases.java:80)
2020-11-29T04:47:26.6362377Z 	at io.prestosql.sql.planner.assertions.ExpressionVerifier.visitSymbolReference(ExpressionVerifier.java:217)
2020-11-29T04:47:26.6366657Z 	at io.prestosql.sql.planner.assertions.ExpressionVerifier.visitSymbolReference(ExpressionVerifier.java:84)
2020-11-29T04:47:26.6369833Z 	at io.prestosql.sql.tree.SymbolReference.accept(SymbolReference.java:41)
2020-11-29T04:47:26.6371666Z 	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
2020-11-29T04:47:26.6374167Z 	at io.prestosql.sql.planner.assertions.ExpressionVerifier.visitCast(ExpressionVerifier.java:264)
2020-11-29T04:47:26.6377527Z 	at io.prestosql.sql.planner.assertions.ExpressionVerifier.visitCast(ExpressionVerifier.java:84)
2020-11-29T04:47:26.6379635Z 	at io.prestosql.sql.tree.Cast.accept(Cast.java:91)
2020-11-29T04:47:26.6380982Z 	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
2020-11-29T04:47:26.6383849Z 	at io.prestosql.sql.planner.assertions.ExpressionVerifier.visitComparisonExpression(ExpressionVerifier.java:350)
2020-11-29T04:47:26.6388217Z 	at io.prestosql.sql.planner.assertions.ExpressionVerifier.visitComparisonExpression(ExpressionVerifier.java:84)
2020-11-29T04:47:26.6391419Z 	at io.prestosql.sql.tree.ComparisonExpression.accept(ComparisonExpression.java:71)
2020-11-29T04:47:26.6393326Z 	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
2020-11-29T04:47:26.6395529Z 	at io.prestosql.sql.planner.assertions.FilterMatcher.detailMatches(FilterMatcher.java:64)
2020-11-29T04:47:26.6398487Z 	at io.prestosql.sql.planner.assertions.PlanMatchPattern.detailMatches(PlanMatchPattern.java:825)
2020-11-29T04:47:26.6401909Z 	at io.prestosql.sql.planner.assertions.PlanMatchingVisitor.visitPlan(PlanMatchingVisitor.java:123)
2020-11-29T04:47:26.6404583Z 	at io.prestosql.sql.planner.assertions.PlanMatchingVisitor.visitPlan(PlanMatchingVisitor.java:35)
2020-11-29T04:47:26.6406818Z 	at io.prestosql.sql.planner.plan.PlanVisitor.visitFilter(PlanVisitor.java:34)
2020-11-29T04:47:26.6408517Z 	at io.prestosql.sql.planner.plan.FilterNode.accept(FilterNode.java:72)
2020-11-29T04:47:26.6410875Z 	at io.prestosql.sql.planner.assertions.PlanMatchingVisitor.matchSources(PlanMatchingVisitor.java:185)
2020-11-29T04:47:26.6414035Z 	at io.prestosql.sql.planner.assertions.PlanMatchingVisitor.visitPlan(PlanMatchingVisitor.java:115)
2020-11-29T04:47:26.6417127Z 	at io.prestosql.sql.planner.assertions.PlanMatchingVisitor.visitPlan(PlanMatchingVisitor.java:35)
2020-11-29T04:47:26.6420015Z 	at io.prestosql.sql.planner.plan.PlanVisitor.visitOutput(PlanVisitor.java:49)
2020-11-29T04:47:26.6422158Z 	at io.prestosql.sql.planner.plan.OutputNode.accept(OutputNode.java:83)
2020-11-29T04:47:26.6424362Z 	at io.prestosql.sql.planner.assertions.PlanAssert.assertPlan(PlanAssert.java:48)
2020-11-29T04:47:26.6426663Z 	at io.prestosql.sql.planner.assertions.PlanAssert.assertPlan(PlanAssert.java:43)
2020-11-29T04:47:26.6428968Z 	at io.prestosql.sql.planner.assertions.PlanAssert.assertPlan(PlanAssert.java:37)
2020-11-29T04:47:26.6431397Z 	at io.prestosql.sql.planner.assertions.BasePlanTest.lambda$assertPlanWithSession$3(BasePlanTest.java:173)
2020-11-29T04:47:26.6433775Z 	at io.prestosql.transaction.TransactionBuilder.execute(TransactionBuilder.java:150)
2020-11-29T04:47:26.6435950Z 	at io.prestosql.testing.LocalQueryRunner.inTransaction(LocalQueryRunner.java:655)
2020-11-29T04:47:26.6456125Z 	at io.prestosql.sql.planner.assertions.BasePlanTest.assertPlanWithSession(BasePlanTest.java:171)
2020-11-29T04:47:26.6459333Z 	at io.prestosql.sql.planner.assertions.BasePlanTest.assertPlan(BasePlanTest.java:112)
2020-11-29T04:47:26.6461829Z 	at io.prestosql.sql.planner.TestUnwrapCastInComparison.testNoUnwrap(TestUnwrapCastInComparison.java:548)
2020-11-29T04:47:26.6465862Z 	at io.prestosql.sql.planner.TestUnwrapCastInComparison.testCastTimestampToTimestampWithTimeZone(TestUnwrapCastInComparison.java:452)
2020-11-29T04:47:26.6469703Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-11-29T04:47:26.6530548Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-11-29T04:47:26.6533320Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-11-29T04:47:26.6535355Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-11-29T04:47:26.6537203Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2020-11-29T04:47:26.6539205Z 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
2020-11-29T04:47:26.6540691Z 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2020-11-29T04:47:26.6542297Z 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2020-11-29T04:47:26.6600761Z 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2020-11-29T04:47:26.6603038Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2020-11-29T04:47:26.6605121Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2020-11-29T04:47:26.6607516Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2020-11-29T04:47:26.6608992Z 	at java.base/java.lang.Thread.run(Thread.java:834)
2020-11-29T04:47:26.6610802Z 	Suppressed: java.lang.Exception: Query: SELECT * FROM (VALUES CAST(NULL AS timestamp(0))) t(a) WHERE a > TIMESTAMP '2020-03-29 01:00:00 UTC'
2020-11-29T04:47:26.6621135Z 		at io.prestosql.sql.planner.TestUnwrapCastInComparison.testNoUnwrap(TestUnwrapCastInComparison.java:555)
2020-11-29T04:47:26.6623264Z 		... 14 more
```

--- a/presto-main/src/test/java/io/prestosql/sql/planner/TestUnwrapCastInComparison.java
+++ b/presto-main/src/test/java/io/prestosql/sql/planner/TestUnwrapCastInComparison.java
@@ -549,7 +549,7 @@ public class TestUnwrapCastInComparison
                     session,
                     output(
                             filter(format("CAST(a AS %s) %s", expectedCastType, inputPredicate),
-                                    values("A"))));
+                                    values("a"))));
         }
         catch (Throwable e) {
             e.addSuppressed(new Exception("Query: " + sql));

